<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Vogt</author>
 <class>Arinc665Qt::Media::MediaSetWidget</class>
 <widget class="QWidget" name="Arinc665Qt::Media::MediaSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>705</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mediaSetGroupBox">
     <property name="title">
      <string>Media Set</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="lblDefaultMediumNumber">
        <property name="text">
         <string>Default Medium Number</string>
        </property>
        <property name="buddy">
         <cstring>defaultMediumNumber</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="contentPage" native="true">
         <attribute name="title">
          <string>Content</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblContent">
            <property name="text">
             <string>Content</string>
            </property>
            <property name="buddy">
             <cstring>content</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="content">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblLoads">
            <property name="text">
             <string>Loads</string>
            </property>
            <property name="buddy">
             <cstring>loads</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="loads">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBatches">
            <property name="text">
             <string>Batches</string>
            </property>
            <property name="buddy">
             <cstring>batches</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="batches">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="checkValueTypePage" native="true">
         <attribute name="title">
          <string>Check Value Types</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="mediaSetCheckValueTypeGroupBox">
            <property name="title">
             <string>Media Set Check Value Type</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QComboBox" name="mediaSetCheckValueType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="listOfFilesCheckValueTypeGroupBox">
            <property name="title">
             <string>List of Files Check Value Type</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="listOfFilesCheckValueType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="listOfLoadsCheckValueTypeGroupBox">
            <property name="title">
             <string>List of Loads Check Value Type</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="listOfLoadsCheckValueType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="listOfBatchesCheckValueTypeGroupBox">
            <property name="title">
             <string>List of Batches Check Value Type</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QComboBox" name="listOfBatchesCheckValueType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="filesCheckValueTypeGroupBox">
            <property name="title">
             <string>Files Check Value Type</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QComboBox" name="filesCheckValueType"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="partNumberLabel">
        <property name="text">
         <string>Part Number</string>
        </property>
        <property name="buddy">
         <cstring>partNumberLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="defaultMediumNumber">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="partNumberLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>partNumberLineEdit</tabstop>
  <tabstop>defaultMediumNumber</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>content</tabstop>
  <tabstop>loads</tabstop>
  <tabstop>batches</tabstop>
  <tabstop>mediaSetCheckValueTypeGroupBox</tabstop>
  <tabstop>mediaSetCheckValueType</tabstop>
  <tabstop>listOfFilesCheckValueTypeGroupBox</tabstop>
  <tabstop>listOfFilesCheckValueType</tabstop>
  <tabstop>listOfLoadsCheckValueTypeGroupBox</tabstop>
  <tabstop>listOfLoadsCheckValueType</tabstop>
  <tabstop>listOfBatchesCheckValueTypeGroupBox</tabstop>
  <tabstop>listOfBatchesCheckValueType</tabstop>
  <tabstop>filesCheckValueTypeGroupBox</tabstop>
  <tabstop>filesCheckValueType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
